volatility3/framework/symbols/linux/utilities/modules.py

```diff
@@ -684,3 +687,82 @@ def get_requirements(cls):
             )
 
         return reqs
+
+
+class ModuleDisplayPlugin(interfaces.configuration.VersionableInterface):
+    """
+    Plugins that enumerate kernel modules (lsmod, check_modules, etc.)
+    must inherit from this class to have unified output columns across plugins.
+    The constructor of the plugin must call super() with the `implementation` set
+    """
+
+    _version = (1, 0, 0)
```

Comment:
Don't need these local variables, since they should be being inherited anyway (you can if you really want, but the plugins should be defining their own needs

Reply:
Sorry, I realized we version this directly, so it's important for this to have its own version!  5:S